src/docbuilder/rustwide_builder.rs

```diff
@@ -76,9 +77,13 @@ impl RustwideBuilder {
         let is_docker = std::env::var("DOCS_RS_DOCKER")
             .map(|s| s == "true")
             .unwrap_or(false);
-        let workspace = WorkspaceBuilder::new(Path::new(workspace_path), USER_AGENT)
-            .running_inside_docker(is_docker)
-            .init()?;
+        let mut builder = WorkspaceBuilder::new(Path::new(workspace_path), USER_AGENT)
+            .running_inside_docker(is_docker);
+        if let Ok(custom_image) = std::env::var("DOCS_RS_LOCAL_IMAGE") {
```

Comment:
1. I find the name `DOCS_RS_LOCAL_IMAGE` ambiguous. Maybe rename it to `DOCS_RS_RUSTWIDE_IMAGE`?

2. Will the use of this environment variable be documented somewhere?

Reply:
I agree LOCAL_IMAGE is a little ambiguous. I don't like RUSTWIDE_IMAGE because this will not work with docker images from the internet (e.g. `alpine:latest`). Maybe `DOCS_RS_LOCAL_DOCKER_IMAGE`? That seems a little verbose though.

I plan to document this in the wiki. I pushed this to a branch of the docs.rs wiki, but it looks like there's not an easy way to see it online. You can view it locally with `git clone https://github.com/rust-lang/docs.rs.wiki.git && cd docs.rs.wiki && git checkout custom-docker`.

Reply:
I don't mind it being verbose. Another alternative would be `DOCS_RS_LOCAL_BUILD_IMAGE`, since this is the image in which the build happens? Or `DOCS_RS_BUILD_IMAGE_LOCAL_ID` since you are referring to local image.

But you pick whatever you feel most comfortable with, most important for me is that there is some documentation that describes how to use this variable. Which is now covered by the wiki 🙂